shoop/xtheme/models.py

```diff
@@ -34,9 +35,13 @@ def appropriate(self, theme, view_name, draft):
         :type view_name: str
         :param draft: Draft mode flag
         :type draft: bool
+        :param global_type: Boolean indicating global placeholder
+        :type global_type: bool|False
         :return: SavedViewConfig (possibly not saved)
         :rtype: SavedViewConfig
         """
+        if global_type:
+            view_name = XTHEME_GLOBAL_VIEW_NAME
```

Comment:
Isn't the view_name already right since the view_name since this is used inside ViewConfig? Do we really need this?